scripts/db/schema-11.up.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
+
+DROP TABLE "studies";
```

Comment:
we can get away with this right now, but as we move into QA / staging / production we'll need to be much more diligent about preserving user data across migrations.

(just something to think about, nothing actionable here right now.)